src/components/Modal/Modal.module.css

```diff
@@ -37,13 +37,14 @@
   max-height: 300px;
   font-size: var(--font-size-medium);
   text-align: center;
-  overflow-y: scroll;
+  overflow-y: auto;
```

Comment:
Before this change, the vertical scroll bar was always shown, looked weird. Now it's only visible if required.